projects/cm_mcu/MonitorI2CTask.h

```diff
@@ -35,46 +35,51 @@ struct MonitorI2CTaskArgs_t {
   const uint8_t n_commands;          // number of commands
   const int n_values;                // number of results
   const uint8_t n_pages;             // number of pages to loop over
+  const uint16_t selpage_reg;        // register for selecting page
   uint16_t *sm_values;
   tSMBus *smbus;                       // pointer to I2C controller
   volatile tSMBusStatus *smbus_status; // pointer to I2C status
   TickType_t updateTick;               // last update time, in ticks
   SemaphoreHandle_t xSem;              // semaphore for controlling access to device
   uint8_t ffpart_bit_mask;             // this mask is only used for detecting 12-ch 25Gbps on the REV2 board
+  uint32_t present_bit_mask;           // this mask is used for all ffs to detect if it is mounted or not
```

Comment:
not clear if these belong here, in the argument to the monitor task.

Reply:
I think you're right. It doesn't seem to be necessary to have this information in a moni2c struct. I will remove it.

Reply:
Yes, it's part of this struct. It's necessary for [this function](https://github.com/apollo-lhc/cm_mcu/blob/77ff48fb0ce36ae73250b3790da4cc1571f4b68a/projects/cm_mcu/LocalTasks.c#L738-L747).

Reply:
i don't doubt we have a reason to store it. I just don't think it belongs in this struct. Same comment as in other places.

Reply:
Done.